gettsim/data/arbeitsl_geld.yaml

```diff
@@ -126,3 +126,26 @@ agsatz1:
     1984-01-01:
       value: 0.67
       note: Prüfen!
+ag_stundengrenze:
+  name:
+    de: Stundengrenze für Bezug von Arbeitslosengeld
+    en: Working hours limit to receive unemployment benefit
+  description:
+    de: §138 (3) SGB III, früher auch §102 (2) AFG. Grenze, ab der nicht mehr Arbeitslosigkeit unterstellt wird.
+    en:
+  values:
```

Comment:
We will have to reorganize this values structure as pointed out in GEP-3. I will change as soon as the new loading function is there!